File: 4229-trim-trailing-vowels/trim-trailing-vowels.py
class Solution:
    def trimTrailingVowels(self, s: str) -> str:
        ans=s
        l=(len(s)*-1)+(-1)
        for i in range(-1,l,-1):
            if s[i]=='a' or s[i]=='e' or s[i]=='i' or s[i]=='o' or s[i]=='u':
                ans=s[0:i]
            else:
                break
        return ans

File: 4229-trim-trailing-vowels/trim-trailing-vowels.java
class Solution {
    public String trimTrailingVowels(String s) {
        StringBuilder sb=new StringBuilder();
        String vowel="aeiou";
        int k=0;
        int c=0;
        for(int i=s.length()-1;i>=0;i--){
            if(vowel.indexOf(s.charAt(i))!=-1){
                continue;
            }
            else{
                k=i;
                c++;
                break;
            }
        }
        if(c>0) for(int i=0;i<=k;i++) sb.append(s.charAt(i));
        return sb.toString();
    }
}